lib/Basics/process-utils.cpp

```diff
@@ -1445,7 +1454,17 @@ bool TRI_SuspendExternalProcess(ExternalId pid) {
 #ifndef _WIN32
   return 0 == kill(pid._pid, SIGSTOP);
 #else
-  return true;
+  TRI_ERRORBUF;
+  
+  HANDLE processHandle = OpenProcess(PROCESS_ALL_ACCESS, FALSE, pid._pid);
+  bool rc = pfnNtSuspendProcess(processHandle) == 0;
```

Comment:
Use `PROCESS_SUSPEND_RESUME` instead of `PROCESS_ALL_ACCESS`.
Also, you should probably check that `pfnNtSuspendProcess` is not null and that the returned handle is valid (i.e., `!= NULL`). When running under a debugger `CloseHandle` throws an exception if it is called with an invalid handle.